src/style-breadcrumb.scss

```diff
@@ -48,18 +47,17 @@
   }
 
   a {
-    color: var(--ds-color-ui-default-default, $ds-color-ui-default-default);
     text-decoration: none; // stylelint-disable-line no-eol-whitespace
   }
 
   @include auro_breakpoint($max: $ds-grid-breakpoint-sm) {
     // Displays left chevron icon
-    ::slotted(auro-icon[name='chevron-left']) {
+    ::slotted([auro-icon][name='chevron-left']) {
```

Comment:
why can't we just do this off the name attribute selector without including `[auro-icon]`? Selectors with multiple identifiers are slower.